sites/hashdev/src/_pages/blog/0002_open-source.mdx

```diff
@@ -14,14 +14,14 @@ At HASH, we publish as much of our code as possible in public.
 
 Depending on the code in question, it'll typically be released under one of three types of license, each of which has slightly different terms attached to it governing what you can and can't do with the underlying source:
 
-1.  **Open-source and permissive licenses:** in essence these licenses give people the freedom to do pretty much whatever they like, including extending the code and keeping it private (making this the most flexible category of license). The entirety of the [Block Protocol](https://github.com/blockprotocol/blockprotocol) is available under these terms (by way of the _MIT License_), alongside most of our blocks and connectors.
-1.  **Open-source and copyleft licenses:** these again give people the freedom to use the code however they like, provided that if they extend or modify the code they contribute those changes back into the open source as well under the same terms. Our [HASH](https://hash.ai/platform/hash) application is available under the _AGPL_ which guarantees this freedom.
+1.  **Open-source and permissive licenses:** in essence these licenses give people the freedom to do pretty much whatever they like, including extending the code and keeping it private (making this the most flexible category of license). The entirety of the [Block Protocol](https://github.com/blockprotocol/blockprotocol) is available under these terms (by way of the _MIT License_), alongside most of our blocks and connectors. Some of our packages like [`error-stack`](https://github.com/hashintel/hash/tree/main/packages/libs/error-stack) and [`deer`](https://github.com/hashintel/hash/tree/main/packages/libs/deer) are dual-licensed under multiple permissive OSS licenses (i.e. both the _MIT License_ and _Apache 2.0 License_).
```

Comment:
```suggestion
1.  **Open-source and permissive licenses:** in essence these licenses give people the freedom to do pretty much whatever they like, including extending the code and keeping it private (making this the most flexible category of license). The entirety of the [Block Protocol](https://github.com/blockprotocol/blockprotocol) is available under these terms (by way of the _MIT License_), alongside most of our blocks and connectors. Some of our packages like [`error-stack`](https://github.com/hashintel/hash/tree/main/packages/libs/error-stack) and [`deer`](https://github.com/hashintel/hash/tree/main/packages/libs/deer) are dual-licensed under multiple permissive OSS licenses (i.e. both the _MIT License_ and _Apache 2.0 License_) at your option.
```

Reply:
```suggestion
1.  **Open-source and permissive licenses:** in essence these licenses give people the freedom to do pretty much whatever they like, including extending the code and keeping it private (making this the most flexible category of license). The entirety of the [Block Protocol](https://github.com/blockprotocol/blockprotocol) is available under these terms (by way of the _MIT License_), alongside most of our blocks and connectors. Some of our packages like [`error-stack`](https://github.com/hashintel/hash/tree/main/packages/libs/error-stack) and [`deer`](https://github.com/hashintel/hash/tree/main/packages/libs/deer) are released under multiple permissive licenses (i.e. incorporable under both _MIT_ and _Apache 2.0_ licenses, at your option).
```

Reply:
Elected for a slight rephrase instead, but thanks for flagging this.

Reply:
Sounds good 👍🏻